configure.ac

```diff
@@ -3827,14 +3827,16 @@ if test "x$enable_llvm" = "xyes"; then
        LLVM_CFLAGS="-I$with_llvm/include -DNDEBUG -D__NO_CTYPE_INLINE -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS"
        LLVM_CXXFLAGS="$LLVM_CFLAGS -std=gnu++11 -fvisibility-inlines-hidden -fno-rtti -Woverloaded-virtual -Wcast-qual"
        LLVM_LDFLAGS="-L$with_llvm/lib"
-       LLVM_SYSTEM_LIBS="-lshell32 -lpsapi -limagehlp -ldbghelp -lm"
+       LLVM_SYSTEM_LIBS="-lshell32 -lpsapi -limagehlp -ldbghelp -lm -lz -luuid -lole32"
        LLVM_LIBS="-lLLVMLTO -lLLVMObjCARCOpts -lLLVMLinker -lLLVMipo -lLLVMVectorize -lLLVMBitWriter \
-         -lLLVMTableGen -lLLVMDebugInfo -lLLVMOption -lLLVMSelectionDAG -lLLVMAsmPrinter \
-         -lLLVMMCDisassembler -lLLVMJIT -lLLVMAnalysis -lLLVMTarget \
-         -lLLVMIRReader -lLLVMAsmParser -lLLVMLineEditor -lLLVMMCAnalysis -lLLVMInstrumentation \
-         -lLLVMInterpreter -lLLVMCodeGen -lLLVMScalarOpts -lLLVMInstCombine -lLLVMTransformUtils \
-         -lLLVMipa -lLLVMAnalysis -lLLVMProfileData -lLLVMMCJIT -lLLVMTarget -lLLVMRuntimeDyld \
-         -lLLVMObject -lLLVMMCParser -lLLVMBitReader -lLLVMExecutionEngine -lLLVMMC -lLLVMCore \
+         -lLLVMTableGen  -lLLVMOption -lLLVMSelectionDAG -lLLVMAsmPrinter \
+         -lLLVMARMAsmPrinter -lLLVMX86AsmParser \
```

Comment:
Why are these 2 needed when we already pass them at https://github.com/mono/mono/pull/10402/commits/8cd8c3ee378a5a474b235016a26617924e3cdb75#diff-67e997bcfdac55191033d57a16d1408aR3758 and https://github.com/mono/mono/pull/10402/commits/8cd8c3ee378a5a474b235016a26617924e3cdb75#diff-67e997bcfdac55191033d57a16d1408aR3754 respectively?

Reply:
These defined were found by removing the flags that llvm-config didn't support, looking for the missing symbols, grepping for their definitions in the `.a` files, and including their names here. I'm not sure why our llvm blob had undefined references to these symbols if we were passing them above. The above seems to be host-specific, I saw the relevant failure when building MinGW x64.